app/helpers/view_helper.rb

```diff
@@ -8,6 +8,7 @@ module ViewHelper
   MAX_NAME_LEN = 255
   MAX_DESC_LEN = 255
   MAX_HOSTNAME_LEN = 255
+  MAX_DESC_LEN_SHORT = 50
```

Comment:
Is this name descriptive enough? Out of context I wouldn't really understand what this is :confused:

Reply:
well to me that made sense, but if we are nitpicking all variable names here, i would go with `MAX_LEN_SMALL_TEXT_FIELD` makes sense that is the length of a small text field in the form

Reply:
I just thought that you'd want to use this constant on this single form, but now I understand. It can be either way I guess.